api/routes/api_games.py

```diff
@@ -17,21 +18,24 @@ def handle_db_timeout_error(e):
 
 @games_app.route('/', methods=['GET'])
 @cache.cached(60*1.3)
-def get_stream_info_today_endpoint():
+def get_stream_info_for_main1_endpoint():
     try:
-        games = get_stream_info_today(session)
-        return get_good_response(games)
+        dto = GamesDTO()
+        games = get_games_today(session, dto)
+        return games
     except Exception as e:
         response = {"error in service": str(e)}
         return get_error_response(response, 500)
 
 
-@games_app.route('/<sport_type>', methods=['GET'])
-@cache.cached(60*1.3, key_prefix=get_cache_key)
-def get_sport_stream_info_today_endpoint(sport_type):
+@games_app.route('/filter', methods=['POST'])
```

Comment:
Not particularly truthful, regarding the end-user. We say it's a "filter" andpoint, when it actually is "games_today" endpoint.

BTW:
Would be nice to have an ACTUAL filtering.
Some global class that would take a query and apply it to the given entity?